### DISCUSSION TITLE
File upload does not work for image with long filename

### DISCUSSION THREAD

**Discussion by thesubhendu** [livewire/livewire#8538] [7. Bugs (archived)]
### Livewire version

^3.4

### Laravel version

v11.10

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Rename your any image file to `image_6638f9f140f2a_rotated_screenshot_2024_04_30_at_9_44_28_am_2024_04_30_8e4379e5407b2febee9a945974d30542_2024_05_06_74a6cdf202fe3f5801b1461835c19097-2024-05-07-1644d482bade289af0e257f762da9d09.png`
2. Try to upload the file, it gives error displayed in image, whereas if I upload file with shorter name it works

<img width="1232" alt="image" src="https://github.com/livewire/livewire/assets/16435611/c6c44310-d787-4b4a-9855-dfd062c0794c">

### Repository or Wirebox URL

livewire.com

### Additional information

Issue I am facing is same as https://github.com/livewire/livewire/issues/2834
This issue is closed as completed but I still face same issue.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Naoray** [comment]
solved in https://github.com/livewire/livewire/pull/9172
